src/test/java/com/woowacourse/pelotonbackend/rider/application/RiderServiceTest.java

```diff
@@ -28,7 +35,22 @@ void setUp() {
     @Test
     void createTest() {
         given(riderRepository.save(any())).willReturn(RiderFixture.createMockRider());
-        riderService.create(RiderFixture.createMockRequest());
+        riderService.create(MemberFixture.createWithId(1L), RiderFixture.createMockRequest());
         verify(riderRepository).save(any());
     }
+
+    @DisplayName("아이디로 Rider를 조회한다.")
+    @Test
+    void findById() {
+        final Rider expectedRider = RiderFixture.createRiderWithId(RiderFixture.TEST_RIDER_ID);
+        when(riderRepository.findById(anyLong())).thenReturn(Optional.of(expectedRider));
+
+        final RiderResponse retrieveRider = riderService.retrieve(1L);
+
+        assertAll(
+            () -> assertThat(retrieveRider.getMemberId()).isEqualTo(expectedRider.getMemberId().getId()),
+            () -> assertThat(retrieveRider.getRaceId()).isEqualTo(expectedRider.getRaceId().getId()),
+            () -> assertThat(retrieveRider).isEqualToIgnoringGivenFields(expectedRider, "raceId", "memberId")
```

Comment:
isEqualToOnlyGivenFields를 이용해서 테스트하는 거 어떨까요 위에 두 줄도 지울 수 있을 것 같아요
```suggestion
            () -> assertThat(retrieveRider).isEqualToComparingOnlyGivenFields(expectedRider, "id", "raceId", "memberId")
```